src/Plugin/EntityReferenceSelection/SiteManagerTrait.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Drupal\og_sm\Plugin\EntityReferenceSelection;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\og\OgGroupAudienceHelperInterface;
+use Drupal\og_sm\SiteManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Trait to inject the site manager in an entity reference selection handler.
+ */
+trait SiteManagerTrait {
+
+  /**
+   * The site manager.
+   *
+   * @var \Drupal\og_sm\SiteManagerInterface
+   */
+  protected $siteManager;
+
+  /**
+   * Constructs a NodeSelection object.
```

Comment:
NodeSelection is hier mss niet altijd van toepassing

Reply:
```suggestion
   * Class constructor.
```